frontend/src/models/api/Note.ts

```diff
@@ -9,6 +9,8 @@ export interface Api_EntityNote extends Api_ConcurrentVersion {
 export interface Api_Note extends Api_ConcurrentVersion {
   id?: number;
   note?: string;
+  appCreateTimestamp?: string;
+  appLastUpdateUserid?: string;
```

Comment:
We already have an interface that you can extend to get those fields. It is called **Api_AuditFields**